### PR TITLE
Changelog for v1.7.2

### PR DESCRIPTION
**Pull request by dennisreimann**
Prepared changelog up to 158e613e29d027db53780513ce9d686e94a449b7

### CODE REVIEW

**Comment by pavlenex** [comment]
Should we aim to have more bug fixes in this release, only 1 bug so far is quite low comparing it to our logged bugs, let's discuss and prioritize bugs to be included in 1.7.2 on the next dev call.

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+### Bug fixes
+
+* UI: Fix store selector transition (#4400) @dennisreimann
+
+### Improvement
+
+* Adapt LN payouts to handle unknown status (#4382) @Kukks
+* Upgrade Bootstrap to v5.2.3; Design System improvements (#4380 #4409) @dennisreimann
+* Wallet import: Surface detailed error messages (#4392) @dennisreimann
+* Bump clightning (#4406) @NicolasDorier
```
Comment (NicolasDorier) [inline Changelog.md:20]: you can remove this, it's only in the tests, the other lines below cover what I changed.

**Comment by NicolasDorier** [comment]
https://github.com/btcpayserver/btcpayserver/pull/4390

**Comment by NicolasDorier** [comment]
https://github.com/btcpayserver/btcpayserver/pull/4390

**Comment by NicolasDorier** [comment]
https://github.com/btcpayserver/btcpayserver/pull/4429

**Comment by NicolasDorier** [comment]
https://github.com/btcpayserver/btcpayserver/pull/4434
https://github.com/btcpayserver/btcpayserver/pull/4432
https://github.com/btcpayserver/btcpayserver/pull/4430
https://github.com/btcpayserver/btcpayserver/pull/4433

**Comment by pavlenex** [comment]
Seems only https://github.com/btcpayserver/btcpayserver/issues/4393 is left for the 1.7.2?

**Comment by dennisreimann** [comment]
@pavlenex Yes, I just checked and closed #4298 and will now try to debug the NFC issue, because kukks is busy with other things.

We could possibly also include #4435 — waiting for @bolatovumar to do a final check, it's good to go from my side.

Also #4431, if @dstrukt can deliver icons today.

imho those shouldn't hold up the release though — let's see if we can include them and aim for tomorrow to ship v1.7.2.

**Comment by pavlenex** [comment]
@dennisreimann Seems Kukss attempted to fix and hopefully fixed it here https://github.com/btcpayserver/btcpayserver/commit/5b7b217b9c7b2e01bef323ff9fb5a9594ac15ecf

**Comment by dennisreimann** [comment]
Yup, v1.7.2 here we come - LFG!

**Comment by bolatovumar** [comment]
@dennisreimann tested #4435 and it looks good!

**Comment by NicolasDorier** [comment]
https://github.com/btcpayserver/btcpayserver/pull/4441

**Comment by dennisreimann** [comment]
We are ready to roll :D

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+* Upgrade Bootstrap to v5.2.3; Design System improvements (#4380 #4409) @dennisreimann
+* Wallet import: Surface detailed error messages (#4392) @dennisreimann
+* Local file system storage as default (#4386) @dennisreimann
+* Remove system plugins from the plugin list (#4429) @NicolasDorier
+* Allow more then 20 accounts when using BTCPayServer.Vault (#4430) @dennisreimann
+* Add BTCTurk rate provider (#4433) @NicolasDorier
+* Rate provider: Use better default than Coingecko when creating a new store (#4416) @NicolasDorier
+* Add DefaultDescription to LNURL withdrawal request (#4434) @bolatovumar
+* Disabled amount/currency update for payment request with active invoices (#4390) @bolatovumar
+* UI: Theme extensions (#4398) @dennisreimann
```
Comment (NicolasDorier) [inline Changelog.md:31]: It seems to me a new feature?

**Pull request closed without merging**